consensus/src/simplex/scheme/bls12381_threshold.rs

```diff
@@ -667,24 +685,21 @@ impl<P: PublicKey, V: Variant + Send + Sync> certificate::Scheme for Scheme<P, V
                     Subject::Nullify { round } => round.encode(),
                 };
 
-                messages.push((Some(seed_namespace.as_slice()), seed_message));
-                signatures.push(&certificate.seed_signature);
-                seeds.insert(context.view(), &certificate.seed_signature);
+                entries.push((
+                    Some(seed_namespace.clone()),
+                    seed_message,
+                    certificate.seed_signature,
+                ));
+                seeds.insert(context.view(), certificate.seed_signature);
             }
         }
 
-        // Aggregate signatures
-        let signature = aggregate_signatures::<V, _>(signatures);
-        aggregate_verify_multiple_messages::<V, _>(
-            identity,
-            &messages
-                .iter()
-                .map(|(namespace, message)| (namespace.as_deref(), message.as_ref()))
-                .collect::<Vec<_>>(),
-            &signature,
-            1,
-        )
-        .is_ok()
+        let entries_refs: Vec<_> = entries
```

Comment:
nit: add a comment that we care about the correctness of individual signatures so we don't use aggregate verification